Calculator/README.md

```diff
@@ -27,3 +27,9 @@ Dividing by Zero will show "Infinity".
 The square root of negative numbers don't exist.
 
 Using parentesis will change the order in which the equation will be processed.
+
+## WIP
+
+-The scientific mode for the calculator is not yet available for this version and so the button currently does nothing.
+
+-Parenthesis will only display for negative numbers to comply with the rule of operators shouldn't be next to eachother. In the future there may be a button for specifically using paranthesis while complying with the priority rule when doing an operation.
```

Comment:
Nice! Maybe you could add a line about the square root function, also, and write down perspectives for the next challenge (which will be based on this one, by refactoring what you have to React 🚀 ) For example, try to describe how interacting with the flask button should give access to more advanced functions in the calculator, would it be a modal? maybe it should extend the current container to the side and add the buttons there? What do you have in mind?